### ISSUE TITLE
Where's the wiki?

### ISSUE THREAD

**Issue by ImportTaste** [KevinParnell/OneeChan#45]
README.md says to refer to the wiki for instructions, but there is no wiki.

**Comment by KevinParnell** [comment]
Yeah, I'm not sure what happened to the original wiki as it wasn't there when I forked it from the closed project.
